cl/favorites/models.py

```diff
@@ -125,6 +138,10 @@ class Meta:
         index_together = (("user", "name"),)
 
 
+@pghistory.track(
+    pghistory.Snapshot(),
+    obj_field=pghistory.ObjForeignKey(related_name="history"),
```

Comment:
If we don't do this, the name is `event`, right? I think I prefer `history` too, but I'm not sure it's worth the cruft and the need to remember to break the default whenever we do this. I also like keeping with defaults because it makes understanding online guides easier.

If it's just event vs history and event is the default, I'd say we go with that. If there's something more here, let me know.

Reply:
Yes, the default related name is "event", so i'll leave them with the default value